main.c

```diff
@@ -96,7 +97,7 @@ main(int argc, char **argv)
 #ifdef DEBUG
 	while ((c = getopt(argc, argv, "dyctvnewVQ:o:")) != EOF) {
```

Comment:
debug mode doesnt have case insensitive mode ?

Reply:
That's why I subscribe to DRY, repetition is the mother of several mistakes/errors.
Fixed by making it DRY here  https://github.com/mingodad/lex/commit/9e69818e9bd4a7b224d6745d800228acabffe9fc .
Thanks for pointing it out.